jobbergate-api/jobbergate_api/config.py

```diff
@@ -32,13 +32,19 @@ class Settings(BaseSettings):
 
     LOG_LEVEL: LogLevelEnum = LogLevelEnum.INFO
 
-    # Database settings
-    DATABASE_HOST: Optional[str]
-    DATABASE_USER: Optional[str]
-    DATABASE_PSWD: Optional[str]
-    DATABASE_NAME: Optional[str]
+    # Database settings  # Default to values from docker-compose.yml
+    DATABASE_HOST: Optional[str] = "db"
+    DATABASE_USER: Optional[str] = "jobbergate"
+    DATABASE_PSWD: Optional[str] = "local-pswd"
+    DATABASE_NAME: Optional[str] = "jobbergate"
     DATABASE_PORT: Optional[int] = 5432
-    DATABASE_URL: Optional[str]
+
```

Comment:
what's the benefit on building the URL by pieces? I guess it goes the other way around what we've been doing on the others APIs

Reply:
not that I will push hard against this way but I'd like some pattern being applied across our code

Reply:
Because it allows you to use the same environment variables to start the docker databases with docker-compose.yaml or the dev cli. For example, if you set your `DATABASE_USER`, `DATABASE_PSWD`, `DATABASE_NAME` in your environment, then `docker-compose.yaml` can spin the database up with those values as well as your local app using them.

Reply:
what I mean is just to use the compose, nothing more. Otherwise, only complexity is added. It is quite complicated to be dropped on a project and to have to set a bunch of "no needed" vars

Reply:
The idea is to have the defaults be usable immediately without having to use docker-compose or create a .env file. for example, with `dev-tools`, I can simply clone the repo and run these three commands:
* `poetry install`
* `poetry run dev-tools db start-test`
* `poetry run pytest`

docker-compose can be helpful and convenient, but I don't like having projects require it to do meaningful things.